metocean_api/ts/read_metno.py

```diff
@@ -318,6 +319,113 @@ def __combine_temporary_files(ts: TimeSeries, save_csv, save_nc, use_cache,tempf
 
     return df
 
+def nora3_wave_spec(ts: TimeSeries, save_csv=False, save_nc=False, use_cache=False):
+    """
+    Extract NORA3 wave spectra timeseries.
+    """
+    ts.variable.append("longitude")  # keep info of regular lon
+    ts.variable.append("latitude")  # keep info of regular lat
+    dates = get_dates(ts.product, ts.start_time, ts.end_time)
+    tempfiles = get_tempfiles(ts.product, ts.lon, ts.lat, dates)
+
+    selection = None
+    lon_near = None
+    lat_near = None
+
+    # extract point and create temp files
+    for i in range(len(dates)):
+        url = get_url_info(ts.product, dates[i])
+
+        if i == 0:
+            station, lon_near, lat_near = get_near_coord(
```

Comment:
@efvik: lon_near and lat_near are not used in the csv-file but rather ds["longitude"][0].values). This breaks the pattern of the other output a bit. Perhaps a comment should be added here to explain why the first coordinate is used instead of the station coordinate

Reply:
ds["longitude"][0].values should be equal to lon_near when the correct station has been selected, but we can use lon_near and lat_near since it is clearer. 

In investigating this I found an issue with the indexing of stations, which are one-indexed. I fixed this by adding 1 to the station number in aux_funcs. See PR #40